package.json

```diff
@@ -18,22 +18,14 @@
   "author": "Suraj Tiwari <surajtiwari020@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "react": "*",
-    "react-native": "*"
+    "react-native": "^0.57.4"
```

Comment:
Any reason behind 0.57.4? Many people still use 0.55.x, it would be good to support them also.